mandatory/4-tax.js

```diff
@@ -17,7 +24,15 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function addTaxAndFormatCurrency() {}
+function addTaxAndFormatCurrency(num,product) {
+  let tax=0.20;
+  let total=(num*product)*(1+tax);
```

Comment:
Once you amend the `calculateSalesTax` function as described above, you can replace this calculation with a call to the `caclulateSalesTax` function.